views/partials/sticky-table-page-size.ejs

```diff
@@ -0,0 +1,14 @@
+var page_size = 10; //Default size
```

Comment:
I thought variables were scoped to a single <script> block. Turns out they aren't. Switching to .js files - thanks.